apps/src/applab/applab.js

```diff
@@ -761,6 +761,10 @@ Applab.init = function(config) {
   Applab.clearEventHandlersKillTickLoop();
   skin = config.skin;
   level = config.level;
+  user = {
+    applabUserId: config.applabUserId,
+    isAdmin: !!config.isAdmin
```

Comment:
If isAdmin came down as "false" (instead of false), this would get us into trouble. I don't think it should, but may be worth double checking, or explicitly check that config is null or undefined.